packages/wallets/src/components/WalletListCardDetails/WalletListCardDetails.tsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
+import { Trans } from 'react-i18next';
+import { useActiveWalletAccount } from '@deriv/api-v2';
+import { WalletText } from '../Base';
 import WalletListCardActions from '../WalletListCardActions/WalletListCardActions';
 import { WalletListCardBalance } from '../WalletListCardBalance';
 import WalletListCardDropdown from '../WalletListCardDropdown/WalletListCardDropdown';
 import './WalletListCardDetails.scss';
 
-const WalletListCardDetails = () => (
-    <div className='wallets-list-details__container'>
-        <WalletListCardDropdown />
-        <div className='wallets-list-details__content'>
-            <WalletListCardBalance />
-            <WalletListCardActions />
+const WalletListCardDetails: React.FC = () => {
+    const { data: activeWallet } = useActiveWalletAccount();
+    const isDemo = activeWallet?.is_virtual;
```

Comment:
Here too; if i get it correctly, `isDemo` might be falsy if the acc is not yet loaded

Reply:
Added isLoading check